bayesian_cnn_prometheus/config.json

```diff
@@ -44,17 +38,8 @@
   "epochs": 3,
   "validation_steps": 5,
   "initial_epoch": 0,
-  "initial_learning_rate": 0.001,
-  "lr_decay_start_epoch": 10,
   "batch_size": 32,
-  "num_gpus": 1,
-  "mc_samples": 3,
-  "border_trim": 0.1,
-  "lower_percentile": 30,
-  "upper_percentile": 70,
-  "chunk_size": [
-    64,
-    64,
-    32
-  ]
```

Comment:
Warto by było to przywrócić, ale zrobię to kiedy będę robił zadanie na porządkowanie config'a, ponieważ teraz to jest trochę myłące. (przynajmniej dla mnie)